application/helpers/filebin_helper.php

```diff
@@ -1,5 +1,30 @@
 <?php
 
+function expiration_duration($duration)
+{
+	$total = $duration * 60 * 60 * 24;
```

Comment:
I haven't checked how many changes this would require, but you could change the input to be in seconds directly. If you haven't looked into that yet, I'll check if this should be done or not. Calculating seconds from the double value might lead to incorrect values.

Reply:
Looks like this requires only a single change in the controller that calls this view. Please change it there so you get seconds directly.